scripts/fbi/crime/preprocess.py

```diff
@@ -258,8 +258,6 @@ def _clean_crime_file(f_input, f_output):
         if field[_STATE_INDEX]:
             # Remove numeric values from state names (comes from footnotes)
             state = _remove_digits(field[_STATE_INDEX])
-            if year == "2016":
```

Comment:
Why this diff?

Reply:
It was missed in previous PR during merge. 
Since 2016 table was updated with the right one, this is no longer needed.